ROMFS/px4fmu_common/init.d/rc.gnd_defaults

```diff
@@ -0,0 +1,36 @@
+#!nsh
+
+set VEHICLE_TYPE ugv
+
+if [ $AUTOCNF == yes ]
+then
+#
+# Default parameters for FW
+#
+	param set RTL_RETURN_ALT 100
+	param set RTL_DESCEND_ALT 100
+	param set RTL_LAND_DELAY -1
+
+	# FW uses L1 distance for acceptance radius
+	#  set a smaller NAV_ACC_RAD for vertical acceptance distance
+	param set NAV_DLL_ACT 0
+	param set NAV_ACC_RAD 2.0
+	param set NAV_LOITER_RAD 5
+
+	param set MIS_LTRMIN_ALT 0.01
+	param set MIS_TAKEOFF_ALT 0.01
```

Comment:
that means you need to refactor the logic of this. it is ok to base the code in the fw controllers and estimators, but having a ground vehicle means that most of the stuff/logic that is applied to fw won't make sense to ground vehicles.